plexupdate.sh

```diff
@@ -365,6 +365,7 @@ function cleanup {
 	rm /tmp/raw 2>/dev/null >/dev/null
 	rm /tmp/failcause 2>/dev/null >/dev/null
 	rm /tmp/kaka 2>/dev/null >/dev/null
+	rm "${DOWNLOADDIR}/${FILENAME}.sha" 2>/dev/null >/dev/null
```

Comment:
Hmm... we might want to check that this is a file before we try it, I'm worried that you might accidentally delete something you shouldn't. Or better yet, store the sha file as a tmp file instead of bundling it with the downloaded file.